### PR TITLE
check for suitable vagrant setup / tinystage

### PR DESCRIPTION
**Pull request by ryanlerch**
1. checks to see if tinystage is reachable before trying to provision
   the bodhi machine.
2. remove the contidional on the hostmanager. it is pretty much required
   now because of the use of ipsilon tinystage

Signed-off-by: Ryan Lerch <rlerch@redhat.com>

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -57,13 +57,12 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
  # This allows developers to view message queues at http://localhost:15672/
  config.vm.network "forwarded_port", guest: 15672, host: 15672
 
- # This is an optional plugin that, if installed, updates the host's /etc/hosts
+ # This is an plugin that, updates the host's /etc/hosts
```
Comment (abompard) [inline Vagrantfile]: This is **a** plugin :-)

**Review by abompard — CHANGES_REQUESTED** [review]

File: Vagrantfile (hunk 1 above)
Comment (ryanlerch) [reply Vagrantfile]: i pronounce plugin, as `eeep-loog-in` so an fits here.

Comment (ryanlerch) [reply Vagrantfile]: jk. changing it now

Comment (ryanlerch) [reply Vagrantfile]: fixed and force-pushed

File: Vagrantfile [hunk 2]
```diff
@@ -57,13 +57,12 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
  # This allows developers to view message queues at http://localhost:15672/
  config.vm.network "forwarded_port", guest: 15672, host: 15672
 
- # This is an optional plugin that, if installed, updates the host's /etc/hosts
+ # This is n plugin that updates the host's /etc/hosts
```
Comment (abompard) [inline Vagrantfile]: Wait what? :-)

**Review by abompard — CHANGES_REQUESTED** [review]

**Review by abompard — APPROVED** [review]

**Pull request merged**